packages/sdk/chainDeploy/mainnets/bsc.ts

```diff
@@ -300,6 +294,13 @@ const curveV2Pools: CurveV2PoolConfig[] = [
     // eps BUSD jCHF
     lpToken: underlying(assets, assetSymbols["JCHF-BUSD"]),
     pool: "0xBcA6E25937B0F7E0FD8130076b6B218F595E32e2",
+    baseToken: underlying(assets, assetSymbols.BUSD),
```

Comment:
is it safe enough to configure the base token manually in the config? is there an on-chain call that can be used to set the base token in the oracle initialization?

Reply:
I am trying to find that out in the open discord for Curve. So far, nothing concrete. Will dig the code.